tests/foreman/ui/test_contenthost.py

```diff
@@ -161,6 +161,8 @@ def test_positive_end_to_end(session, repos_collection, vm):
     result = vm.run(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
     assert result.status == 0
     with session:
+        session.organization.select(module_org.name)
```

Comment:
Why there is need to select the module_org ? It should be selected by default by as user is got created in conftest.py, I can understand here there may be problem you facing because of module location. there has to be nicer way to fix this issue.

Reply:
I am going to test with just setting 'Default Location'